src/app/home/home.component.scss

```diff
@@ -127,10 +127,10 @@ div.outer-wrapper {
 							pre {
 								margin: 0px 0px;
 								font-family: 'Roboto', sans-serif;
-								max-width: 100px;
+								max-width: 150px;
```

Comment:
Why do we even need a `max-width` here?

Infact, I think we just need `font-family` and `margin`

Reply:
Ummm yes, now when I think about it, it's not relevant here. I will remove it.